samples/apps/bookinfo/install_cluster_extension.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+#
```

Comment:
Okay, we definitely need a folder here. Too many scripts in the root. I asked Greg to move consul stuff to separate folder.

Reply:
Maybe be good to move both consul and 'simple' VMs to same folder ?

Reply:
no. rule formats are different. see bookinfo consul folder

Reply:
Removed this completely - the setup extension instructions will take care of all that is needed, we only need the script 
to setup mysql and whatever app we want to show.

For now I'm keeping details - but I agree a go app will be better ( for 0.3)